src/auth/screens/language-settings.js

```diff
@@ -34,4 +34,9 @@ export default [
     emojiCode: ':flag-ru:',
     name: 'Русский',
   },
+  {
+    code: 'gl',
+    emojiCode: ':flag_white:',
```

Comment:
The list of emoji codes we can use are [here](). You're right, `flag_white` doesn't exist but you can use `:waving_white_flag:` to have this :white_flag: